commitizen/cz/conventional_commits/translation_multilanguage.py

```
@@ -0,0 +1,69 @@
+import os
+
+# mypy: ignore-errors
```

Why do we ignore errors?